src/equations/acoustic_perturbation_2d.jl

```diff
@@ -361,35 +362,41 @@ end
 
 
 # Calculate maximum wave speed for local Lax-Friedrichs-type dissipation
-# TODO: Better comment, here this is NOT really a normal vector
-@inline function max_abs_speed_naive(u_ll, u_rr, normal_vector::AbstractVector, equations::AcousticPerturbationEquations2D)
+@inline function max_abs_speed_naive(u_ll, u_rr, normal_direction::AbstractVector, equations::AcousticPerturbationEquations2D)
   # Calculate v = v_prime + v_mean
-  v_prime_ll = normal_vector[1]*u_ll[1] + normal_vector[2]*u_ll[2]
-  v_prime_rr = normal_vector[1]*u_rr[1] + normal_vector[2]*u_rr[2]
-  v_mean_ll = normal_vector[1]*u_ll[4] + normal_vector[2]*u_ll[5]
-  v_mean_rr = normal_vector[1]*u_rr[4] + normal_vector[2]*u_rr[5]
+  v_prime_ll = normal_direction[1]*u_ll[1] + normal_direction[2]*u_ll[2]
+  v_prime_rr = normal_direction[1]*u_rr[1] + normal_direction[2]*u_rr[2]
+  v_mean_ll = normal_direction[1]*u_ll[4] + normal_direction[2]*u_ll[5]
+  v_mean_rr = normal_direction[1]*u_rr[4] + normal_direction[2]*u_rr[5]
 
   v_ll = v_prime_ll + v_mean_ll
   v_rr = v_prime_rr + v_mean_rr
 
   c_mean_ll = u_ll[6]
   c_mean_rr = u_rr[6]
 
-  λ_max = (max(abs(v_ll), abs(v_rr)) + max(c_mean_ll, c_mean_rr)) * norm(normal_vector)
+  λ_max = (max(abs(v_ll), abs(v_rr)) + max(c_mean_ll, c_mean_rr)) * norm(normal_direction)
 end
 
 
-# This routine was adapted from the compressible EUler equations.
-# TODO: Verify that this wall boundary condition is correct for the APE equations.
-#       In the limit where normal_direction = (±1, 0) or (0, ±1) it is consistent to
-#       `boundary_condition_wall` above
+"""
+   boundary_state_slip_wall(u_inner, normal_direction, equations)
```

Comment:
```suggestion
    boundary_state_slip_wall(u_inner, normal_direction, equations::AcousticPerturbationEquations2D)
```
You need 4 spaces to indicate a code block (check the help printed in the REPL when asking for `?boundary_state_slip_wall`).

Reply:
Done. I will include this with the change to the reference in the docstring